frontend/apps/marketing/.env.example

```diff
@@ -0,0 +1,31 @@
+####################################
+# Contentful Environment Variables #
+####################################
+
+# For values, see: https://www.contentful.com/developers/docs/references/authentication/
+
+# Contentful Space ID
+# Viewable in the Contentful web app under Settings > API keys
+# See: https://contentful.github.io/contentful.js/contentful/latest/interfaces/CreateClientParams.html#space
+CONTENTFUL_SPACE_ID=
```

Comment:
Process Q for after this is merged: do we need to fill these in locally with a `.env` copy of this file? Similar to what we've done before with the `APPLITOOLS_API_KEY` token?

Reply:
Correct - and I'll update the README to include this.